ehitchhiking-ui/src/app/shared/components/rate-passengers-modal/rate-passengers-modal.component.html

```diff
@@ -9,21 +9,28 @@
 				<th class="table-th">Passenger</th>
 				<th class="table-th">Stars</th>
 				<th class="table-th">Rating</th>
+        <th class="table-th">Blacklist</th>
 			</tr>
 		</thead>
 		<tbody>
-			<tr *ngFor="let passenger of passengers; trackBy: trackById">
+    <tr [ngClass]="{'passenger-row-active': passenger.isAddedToBlackList === true}"
```

Comment:
[ngClass]="{'passenger-row-active': passenger.isAddedToBlackList}"
there is no deed to add === true, passenger.isAddedToBlackLis is already boolean

Reply:
reworked